models/public/index.md

```diff
@@ -9,64 +9,64 @@ The models can be downloaded via Model Downloader
+| SE-ResNet 152               | Caffe\*                            | [se-resnet-152](./se-resnet-152/se-resnet-152.md) | 78.506/94.45 | 22.709 | 66.746 |
+| SE-ResNeXt 50               | Caffe\*                            | [se-resnext-50](./se-resnext-50/se-resnext-50.md) | 78.968/94.63 | 8.533 | 27.526|
+| SE-ResNeXt 101              | Caffe\*                            | [se-resnext-101](./se-resnext-101/se-resnext-101.md) | 80.168/	95.19 | 16.054 | 48.886 |
+| SqueezeNet v1.0             | Caffe\*                            | [squeezenet1.0](./squeezenet1.0/squeezenet1.0.md)| 57.684/ 80.38| 1.737 | 1.248 |
+| SqueezeNet v1.1             | Caffe\*<br>Caffe2\*                | [squeezenet1.1](./squeezenet1.1/squeezenet1.1.md)<br>[squeezenet1.1-caffe2](./squeezenet1.1-caffe2/squeezenet1.1-caffe2.md)| 58.382/81<br>56.502/79.576 | 0.785 | 1.236 |
+| VGG 16                      | Caffe\*                            | [vgg16](./vgg16/vgg16.md) | 70.968/89.878 | 30.974 | 138.358 |
+| VGG 19                      | Caffe\*<br>Caffe2\*                | [vgg19](./vgg19/vgg19.md)<br>[vgg19-caffe2](./vgg19-caffe2/vgg19-caffe2.md) | 71.062/89.832<br>71.062/89.832 | 39.3 | 143.667  |
 
 **Octave Convolutions Networks**
 
 This is are modifications of networks using Octave Convolutions. More details can be found [here](https://arxiv.org/abs/1904.05049).
 
 | Model Name                | Implementation | OMZ Model Name | Accuracy | GFlops | mParams |
 | ------------------------- | ---------------| -------------- | -------- | ------ | ------- |
-| DenseNet 121, alpha=0.125 | [MXNet\*](./octave-densenet-121-0.125/octave-densenet-121-0.125.md)   | octave-densenet-121-0.125 | | 4.883 | 7.977 |
-| ResNet 26, alpha=0.25     | [MXNet\*](./octave-resnet-26-0.25/octave-resnet-26-0.25.md)           | octave-resnet-26-0.25     | | 3.768 | 15.99 |
-| ResNet 50, alpha=0.125    | [MXNet\*](./octave-resnet-50-0.125/octave-resnet-50-0.125.md)         | octave-resnet-50-0.125    | | 7.221 | 25.551 |
-| ResNet 101, alpha=0.125   | [MXNet\*](./octave-resnet-101-0.125/octave-resnet-101-0.125.md)       | octave-resnet-101-0.125   | | 13.387 | 44.543 |
-| ResNet 200, alpha=0.125   | [MXNet\*](./octave-resnet-200-0.125/octave-resnet-200-0.125.md)       | octave-resnet-200-0.125   | | 25.407 | 64.667 |
-| ResNeXt 50, alpha=0.25    | [MXNet\*](./octave-resnext-50-0.25/octave-resnext-50-0.25.md)         | octave-resnext-50-0.25    | | 6.444 | 25.02 |
-| ResNeXt 101, alpha=0.25   | [MXNet\*](./octave-resnext-101-0.25/octave-resnext-101-0.25.md)       | octave-resnext-101-0.25   | | 11.521 | 44.169 |
-| SE-ResNet 50, alpha=0.125 | [MXNet\*](./octave-se-resnet-50-0.125/octave-se-resnet-50-0.125.md)   | octave-se-resnet-50-0.125 | | 7.246 | 28.082 |
+| DenseNet 121, alpha=0.125 | MXNet\*        | [octave-densenet-121-0.125](./octave-densenet-121-0.125/octave-densenet-121-0.125.md) | 76.066/93.044 | 4.883 | 7.977 |
```

Comment:
if I right remember, we discussed merging OctaveNets to classification models list

Reply:
Done